crates/federate/src/worker.rs

```diff
@@ -32,10 +32,10 @@ static WORK_FINISHED_RECHECK_DELAY: Duration = Duration::from_secs(1);
 static WORK_FINISHED_RECHECK_DELAY: Duration = Duration::from_secs(30);
 #[cfg(debug_assertions)]
 static FOLLOW_ADDITIONS_RECHECK_DELAY: Lazy<chrono::Duration> =
-  Lazy::new(|| chrono::Duration::seconds(1));
+  Lazy::new(|| chrono::Duration::minutes(1));
```

Comment:
@phiresky I'm not sure what these should be now, but at least in my testing, this fixes the worst of the CPU spikes.

Reply:
Its fine to do this in debug mode in general, but for the federation tests this delay needs to be low or otherwise they will take a long time